### PR TITLE
 Translates the Latest Best Practices and News

### PR DESCRIPTION
**Pull request by marcelosdm**
Hello again guys!

In this PR, I'm translating the **Latest Best Practices and News**, and some other details that I was missing! 😃

### CODE REVIEW

File: README.brazilian-portuguese.md [hunk 1]
```diff
@@ -18,21 +18,21 @@
...
 
-- **New best practice:** 4.2: Include 3 parts in each test name - [_From the section "Testing and overall quality"_](https://github.com/i0natan/nodebestpractices#4-testing-and-overall-quality-practices)
+- **Nova prática recomendada:** 4.2: Inclua 3 partes em cada nome de teste - [_Da seção "Teste e qualidade geral"_](https://github.com/i0natan/nodebestpractices#4-testing-and-overall-quality-practices)
 
-- **New best practice:** 7.1: Prefer native JS methods over user-land utils like Lodash - [_From the section "Performance"_](https://github.com/i0natan/nodebestpractices#7-performance-best-practices)
+- **Nova prática recomendada:** 7.1: Prefira métodos JS nativos ao invés de utilitários de usuário, como o Lodash - [Da seção "Performance"\_](https://github.com/i0natan/nodebestpractices#7-performance-best-practices)
 
-- **News update:** [We kicked-off the performance section, wanna join?](https://github.com/i0natan/nodebestpractices/issues/302)
+- **Novidades:** [Demos o pontapé inicial na seção de performance, quer fazer parte?](https://github.com/i0natan/nodebestpractices/issues/302)
```
Comment (sagirk) [inline README.brazilian-portuguese.md]: Looks like you had format-on-save turned on in your editor while editing this. 

We probably want to undo the changes past this point.

Everything else above looks great! Thanks! 👍

Comment (marcelosdm) [reply README.brazilian-portuguese.md]: @sagirk Sorry!!! You're right: I really had format-on-save turned on.

But, now I have a little doubt (a newbie one): how can I resolve this? I already changed in my local file. Shoul I just push to remote again?

Comment (BrunoScheufler) [reply README.brazilian-portuguese.md]: Either this or you could copy the file's current state (so only the lines below the changes you wanted to make) and simply replace your formatted content to undo those changes easily.

Comment (marcelosdm) [reply README.brazilian-portuguese.md]: Thanks @BrunoScheufler 

I've made those changes and I think we're good to go!

Sorry again for this 😣

File: README.brazilian-portuguese.md [hunk 2]
```diff
@@ -282,15 +282,14 @@ function someFunction()
 const count = 2;
 (function doSomething() {
   // do something amazing
-}());
+})();
```
Comment (sagirk) [inline README.brazilian-portuguese.md]: nit: Looks like this slipped through. Please undo this too.

File: README.brazilian-portuguese.md [hunk 3]
```diff
@@ -282,15 +282,14 @@ function someFunction()
...
 
 // Avoid — throws exception
 const count = 2 // it tries to run 2(), but 2 is not a function
 (function doSomething() {
   // do something amazing
 }())
 ```
-
```
Comment (sagirk) [inline README.brazilian-portuguese.md]: nit: Same as above

File: README.brazilian-portuguese.md [hunk 4]
```diff
@@ -1075,6 +1074,7 @@ Our collaborators are members who are contributing to the repository on a reguar
 
 <br/>
 
+
```
Comment (sagirk) [inline README.brazilian-portuguese.md]: nit: Same as above

File: README.brazilian-portuguese.md (hunk 2 above)
Comment (BrunoScheufler) [reply README.brazilian-portuguese.md]: That one is actually a good change since the previous code wasn't correct 👍

Comment (sagirk) [reply README.brazilian-portuguese.md]: It's only a matter of preference. Both are correct.
```js
(function() {
  //body
}())
```
and 
```js
(function() {
  //body
})()
```
do the same thing.

The only reason I'm hesitating is 'coz if we make this change here, then all the other language READMEs need to be synced too. Also, [L291](https://github.com/i0natan/nodebestpractices/pull/358/files#diff-0dff8ae931792f86c957630a81d0e679R291) will need to be updated to match the same IIFE pattern.

I personally don't find it worth the change, but am not opposed to it if someone wants to update all the change-sites.

Comment (BrunoScheufler) [reply README.brazilian-portuguese.md]: Ah I see, let's change this back then 👍

Comment (sagirk) [inline README.brazilian-portuguese.md]:
```suggestion
}());
```

File: README.brazilian-portuguese.md (hunk 4 above)
Comment (sagirk) [inline README.brazilian-portuguese.md]:
```suggestion
```

File: README.brazilian-portuguese.md [hunk 5]
```diff
@@ -282,15 +282,14 @@ function someFunction()
...
+})();
 
 // Avoid — throws exception
 const count = 2 // it tries to run 2(), but 2 is not a function
 (function doSomething() {
   // do something amazing
 }())
 ```
-
 <br/><br/>
```
Comment (sagirk) [inline README.brazilian-portuguese.md]:
```suggestion

<br/><br/>
```

File: README.brazilian-portuguese.md (hunk 3 above)
Comment (sagirk) [reply README.brazilian-portuguese.md]: Resolved since the suggestion below takes care of it.

File: README.brazilian-portuguese.md (hunk 4 above)
Comment (sagirk) [reply README.brazilian-portuguese.md]: Resolving since the suggestion below takes care of it.

File: README.brazilian-portuguese.md (hunk 2 above)
Comment (sagirk) [reply README.brazilian-portuguese.md]: Resolving since the suggestion below takes care of it.

**Comment by marcelosdm** [comment]
> @marcelosdm To save you some hassle, just commit these suggestions and we're good to go.

@sagirk I'm sorry to be so late to commit! Anyway, I've just did it!

Thanks again friend, and I'll pay attention next time! 😕
